AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -18,13 +18,16 @@ struct ConfirmCreatingSessionView: View {
     }
 
     @State private var isPresentingAlert: Bool = false
-    @State private var didStartRecordingSession = false
     @EnvironmentObject var selectedSection: SelectSection
     @EnvironmentObject private var sessionContext: CreateSessionContext
     @EnvironmentObject private var locationTracker: LocationTracker
     @EnvironmentObject private var tabSelection: TabBarSelection
+    @EnvironmentObject var persistenceController: PersistenceController
+    @EnvironmentObject var bluetoothManager: BluetoothManager
+    @EnvironmentObject var userAuthenticationSession: UserAuthenticationSession
+    @EnvironmentObject private var microphoneManager: MicrophoneManager
```

Comment:
Something we need to address/discuss in the future:
problem with making all those `Manager` type classes an `EnvironmentObject` and importing directly in views is that it will reload the UI way too often - on each change of each of the managers. It will also take the possibility to decide when we want to reload the UI out of our hands. For example importing `LocationTracker` in a view where we just need the one latest location will still cause the UI to reload on each location fetch from `CLLocationManager`. It's pretty bad on its own, but If paired with another high-frequency `Manager` like `MicrophoneManager` those values can get really high and energy impact of the app skyrockets (maybe even UI cluttering). I feel like this `EnvironmentObject` thing is not supposed to be a dependency management tool, rather a way to share UI-related data transfer objects between views.